core/conn/unixodbc/odbc/odbcclient/unixcli/cli/ctosqlconv.cpp

```diff
@@ -3565,12 +4229,12 @@ unsigned long ODBC::ConvertCharToInt64(SQLPOINTER srcDataPtr,
 	int		tempLen;
 	char	cTmpBuf[100];
         bool truncation = false;
-
+    int     nuberLen = 0;
```

Comment:
The variable nuberLen doesn't seem to be used anywhere below. (It doesn't hurt anything though.)